server/migrations/20190603150654-create-article.js

```diff
@@ -0,0 +1,48 @@
+const Articles = {
+  up: (queryInterface, Sequelize) => queryInterface.createTable('Articles', {
+    id: {
+      allowNull: false,
+      autoIncrement: true,
+      primaryKey: true,
+      type: Sequelize.INTEGER
+    },
+    title: {
+      type: Sequelize.STRING,
+      allowNull: false
+    },
+    slug: {
+      type: Sequelize.STRING,
+      allowNull: false
+    },
+    discription: {
```

Comment:
@Eazybee, I think there is a typo here too.

Reply:
Where?  Do you mind being specific

Reply:
```discription``` instead of ```description```

Reply:
ok